src/panels/lovelace/editor/config-elements/hui-area-card-editor.ts

```diff
@@ -0,0 +1,127 @@
+import { HomeAssistant } from "../../../../types";
+import { AreaCardConfig } from "../../cards/types";
+import "../../components/hui-action-editor";
+import "../../components/hui-entity-editor";
+import "../../components/hui-theme-select-editor";
+import { LovelaceCardEditor } from "../../types";
+import { baseLovelaceCardConfig } from "../structs/base-card-struct";
+import { EditorTarget } from "../types";
+import { configElementStyle } from "./config-elements-style";
+
+const cardConfigStruct = assign(
+  baseLovelaceCardConfig,
+  object({
+    area: optional(string()),
+    image: optional(string()),
+    theme: optional(string()),
+  })
+);
+
+@customElement("hui-area-card-editor")
+export class HuiAreaCardEditor
+  extends LitElement
+  implements LovelaceCardEditor
+{
+  @property({ attribute: false }) public hass?: HomeAssistant;
+
+  @state() private _config?: AreaCardConfig;
+
+  public setConfig(config: AreaCardConfig): void {
+    assert(config, cardConfigStruct);
+    this._config = config;
+  }
+
+  get _area(): string {
+    return this._config!.area || "";
+  }
+
+  get _image(): string {
+    return this._config!.image || "";
+  }
+
+  get _theme(): string {
+    return this._config!.theme || "";
+  }
+
+  protected render(): TemplateResult {
+    if (!this.hass || !this._config) {
+      return html``;
+    }
+
+    return html`
+      <div class="card-config">
+        <ha-area-picker
+          .hass=${this.hass}
+          .value=${this._area}
+          .placeholder=${this._area}
+          .configValue=${"area"}
+          .label=${this.hass.localize("ui.dialogs.entity_registry.editor.area")}
+          @value-changed=${this._valueChanged}
+        ></ha-area-picker>
+        <paper-input
+          .label="${this.hass.localize(
+            "ui.panel.lovelace.editor.card.generic.image"
+          )} (${this.hass.localize(
+            "ui.panel.lovelace.editor.card.config.optional"
+          )})"
+          .value=${this._image}
+          .configValue=${"image"}
+          @value-changed=${this._valueChanged}
+        >
+        </paper-input>
+        <hui-theme-select-editor
+          .hass=${this.hass}
+          .value=${this._theme}
+          .configValue=${"theme"}
+          @value-changed=${this._valueChanged}
+        ></hui-theme-select-editor>
+      </div>
+    `;
+  }
+
+  private _valueChanged(ev: CustomEvent): void {
+    if (!this._config || !this.hass) {
+      return;
+    }
+    const target = ev.target! as EditorTarget;
+    const value = ev.detail.value;
+
+    if (this[`_${target.configValue}`] === value) {
+      return;
+    }
+    let newConfig;
+    if (target.configValue) {
+      if (value !== false && !value) {
+        newConfig = { ...this._config };
+        delete newConfig[target.configValue!];
+      } else {
+        newConfig = {
+          ...this._config,
+          [target.configValue!]:
+            target.configValue === "icon_height" && !isNaN(Number(target.value))
```

Comment:
`icon_height`? :D

Reply:
Cleaning up the editor is still on the todo :)